packages/documentation/src/pages/getting-started/common-tasks/new-end-to-end-test.mdx

```diff
@@ -23,124 +24,50 @@ Front end engineers use end-to-end (e2e) tests in `vets-website` to validate mul
-- perform `axeCheck` on the main body of the application on each page - see [Custom Nightwatch commands](#custom-nightwatch-commands)
-- assert that each navigation is successful
+- Separate navigation from field input
+  - Use a **main test file** for navigation, assertions, and calls helpers
+  - Use a **helper file** for filling out forms
+- Create separate, numbered **main test files** to organize tests by their focus:
+  - **00-all-fields.cypress.spec.js** - required and optional fields
+  - **01-required.cypress.spec.js** - only required fields
+  - **02-accessibility.cypress.spec.js** - validates accessibility
+  - **03-auth.cypress.spec.js** - validates authentication
+  - **04-cross-cutting-feature.cypress.spec.js** - validates one feature used across several pages (e.g. save in progress)
+- Group tests by pages and use a comment to indicate what page is being tested
+- Mock all api responses before starting the test. See [Mock API responses](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/testing/end-to-end#mocks)
+- Use `waitForElementVisible` before interacting with any element on the page
+- Use `Timeouts` constants for setting timeouts (`platform/testing/e2e/timeouts.js`)
+- Use helpers for filling data and performing actions on the page
+- Perform `axeCheck` on the main body of the application on each page - see [axeCheck](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/testing/e2e/cypress/support/commands/axeCheck.js)
+- Assert that each navigation is successful
 
 ## Mocking API responses
 
 A mock server runs with the end-to-end tests to allow tests to make production-like calls.
 
-- [`mockData(data, token = nul)`](https://github.com/department-of-veterans-affairs/vets-website/blob/6d97a63bd60d79864661cc757814ca041648d5c9/src/platform/testing/e2e/nightwatch-commands/mockData.js#L12-L14)
-  - custom Nighwatch command that mocks the data at the endpoint provided
-  - mock server is started as part of the end-to-end testing script
+See the [Mocks section of Cypress Resources Reference Guide](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/testing/end-to-end#mocks) for detailed mock API examples currently used.
 
-## Custom Nightwatch commands
+## Custom Cypress commands
 
-Nightwatch supports extending its client api with [custom commands](https://nightwatchjs.org/guide/#writing-custom-commands). Custom commands are located in `src/platform/testing/e2e/nightwatch-commands`
+Cypress supports extending its client api with [custom commands](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/testing/e2e/cypress/support/commands).
 
-_Command commands are available on the Nightwatch client e.g. `client.axeCheck()`. Below are some of the commonly used custom Nightwatch commands._
+Below are some of the commonly used custom Nightwatch commands (accessible from the link above).\_
```

Comment:
Should this say Cypress instead of Nightwatch?

For example:
```
- Below are some of the commonly used custom Nightwatch commands...
+ Below are some of the commonly used custom Cypress commands...
```